### ISSUE TITLE
add text to audio announcements

### ISSUE THREAD

**Issue by RichMorin** [ThePacielloGroup/AT-browser-tests#55]
I am unable to understand the "audio of JAWS announcing test file content" playback. Part of the problem is that I'm unpracticed with parsing rapid speech. Another issue is that I have trouble hearing high frequencies.

I would find it really useful to have the transcribed text, along with the audio recording. Another possibility might be to add a speed control to the playback UI.
